src/documents/css/idg.css.styl

```diff
@@ -499,10 +514,21 @@ contrast-theme(foregroundColor, backgroundColor) {
         stroke: foregroundColor;
         fill: backgroundColor;
     }
-    
+
     .idg-highlight-section {
         border: 0.2rem solid foregroundColor;
     }
+
+    footer svg {
+        stroke: foregroundColor;
+        fill: backgroundColor;
+    }
+
+    /* Special styling for P4All logo */
+    svg #prosperity, svg #tree {
+        stroke: backgroundColor;
+        fill: foregroundColor;
+    }
```

Comment:
This looks similar to styling from previous in the file. Can this duplication be factored out?